content/posts/April-2021.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "April 2021"
+date: 2021-04-09T16:40:16+01:00
+draft: false
+weight: 76
+---
+
+## Week 14 (April 4th - April 9st)
+
+- Honza converted packit's test suite from STI to FMF and configured packit to
+  synchronize the suite to Fedora dist-git
+  ([packit#1192](https://github.com/packit/packit/pull/1192)).
+- Franta fixed a bug in packit which kept only appending targets to an existing
+  COPR project which is no longer a case - dropped targets are now being
+  removed
+  ([packit#1197](https://github.com/packit/packit/pull/1197)).
+- Laura fixed how packit-service pushes alerts to sentry, which should make it
+  more convenient for us, maintainers to know which project is affected by the
+  particular alert
+  ([packit-service#1048](https://github.com/packit/packit/pull/1048)).
```

Comment:
The url is wrong, correct is https://github.com/packit/packit-service/pull/1048
And I'd not put it into the blog, this is not something readers care about.

Reply:
well, I wanted to be transparent; ok, will drop it